tests/unit/checkboxradio/checkboxradio_common.js

```diff
@@ -0,0 +1,15 @@
+TestHelpers.commonWidgetTests( "checkboxradio", {
+	noDefaultElement: true,
```

Comment:
But there is one, `defaultElement: "<input type='checkbox'>",`. Is this outdated and can be dropped?